docs/using/event-types.md

```diff
@@ -47,81 +47,90 @@ have a more specific definition :)
+attributes.
 
-1. Consumers using older schemas can safely read events validated by
-   newer schemas.
-2. Clients using newer schema can safely read events validated by
-   older schemas.
+#### Forward compatible
 
-It's useful for long term data storage and analysis. It guarantees the
-highest quality data for business intelligence.
+The `forward` mode has been designed to allow event type owners to
+expand their schemas without breaking existing consumers.
 
-Only a subset of json schema is allowed, to make sure that no
-accidental incompatibilities or ambiguities are introduced. This mode
-is required for data to be stored and processed by the data lake.
+It's called forward because consumers using older schemas can safely
+read events generated by newer schemas. On the other hand, it's not
+backward compatible, in the sense that reading older events using
+newer schemas is not safe.
+
+For example, if the consumer has been developed with schema version
+`1.5.0` in mind, reading events from version `1.5.0` on is safe, for
+example, version `1.6.0`. But reading event generated by older
+schemas, like `1.4.0`, is not safe.
 
 Supported changes:
 
-1. changes to meta attributes: titles, descriptions and examples.
-2. addition of new optional attributes.
+1. changes to meta attributes: titles and descriptions.
+2. addition of new attributes, both optional and required one.
+3. marking attributes as required.
 
-Unsupported json-schema attributes:
+Under this mode event validation accepts events with fields not
+declared in the schema. In other words, it's not necessary to specify
+`additionalProperties true` in the schema, since this is the default
+behaviour of json-schema validator.
 
-1. additionalProperties
-2. additionalItems
-3. not
-4. patternProperties
+We desencourage the usage of `additionalProperties` entirely. The more
+complete a schema definition the more stable and valuable the
+events. The ability to not specify every attribute is there only for
+some very specific situations when it's not possible to define them
+all.
 
-Removing the support for these features is necessary to avoid the
-introduction of incompatible changes. Since additionalProperties is
-not supported, its value is false by default: all events with
-undeclared attributes will be rejected.
+Consumers reading events from forward compatible event types SHOULD
+ignore event attributes not declared in the event type schema. This is
+aligned with
+[API guidelines for compatibility](http://zalando.github.io/restful-api-guidelines/compatibility/Compatibility.html).
 
-#### none
+#### Incompatible
 
-Schemas can be changed arbitrarily. Nakadi will automatically manage
-the version field so that if an incompatible change happens (MAJOR
-version bump) consumers are aware of it.
+Under compatibility mode `none` schemas can be changed
+arbitrarily. This mode is not recommended unless there is a very good
+reason not to provide any compatibility guarantee.
```

Comment:
In this section I miss 2 informations -- pls. add:

1) What is the default mode? 
--> forward compatibility with "hint: full compatibility usage is definitely encouraged, default mode will change in near future"

2) What Compatibility Mode change paths are supported?